frame/instrumental-strategy-pablo/src/lib.rs

```diff
@@ -209,7 +252,57 @@ pub mod pallet {
 
 	impl<T: Config> Pallet<T> {
 		#[transactional]
-		fn do_rebalance(_vault_id: &T::VaultId) -> DispatchResult {
+		fn do_rebalance(vault_id: &T::VaultId) -> DispatchResult {
+			let asset_id = T::Vault::asset_id(vault_id)?;
+			let account_id = T::Vault::account_id(vault_id);
+			let pool_id = Self::pools(asset_id).ok_or(Error::<T>::PoolNotFound)?;
+			let task = T::Vault::available_funds(vault_id, &Self::account_id())?;
+			match task {
+				FundsAvailability::Withdrawable(balance) => {
+					let vault_account = T::Vault::account_id(vault_id);
+					let lp_token_amount = T::Pablo::amount_of_lp_token_for_added_liquidity(
+						pool_id,
+						T::Balance::zero(),
+						balance,
+					)?;
+					T::Pablo::add_liquidity(
+						&vault_account,
+						pool_id,
+						T::Balance::zero(),
+						balance,
+						lp_token_amount,
+						bool::default(),
```

Comment:
I used `bool::deafult()` because I don't know, what should be here. In many places used `true`, but `bool::default()` return `false`. It's `keep_alive` parameter.

Reply:
I think we should change `bool::default()` to `true` or `false`, because this can be misleading.